packages/react-instantsearch/examples/autocomplete/src/App-Hits-And-Facets.js

```diff
@@ -0,0 +1,143 @@
+import React, {Component} from 'react';
+import {InstantSearch} from 'react-instantsearch/dom';
+import {createConnector} from 'react-instantsearch';
+import {connectSearchBox, connectRefinementList} from 'react-instantsearch/connectors';
+import Autosuggest from 'react-autosuggest';
+import {isEqual, forOwn} from 'lodash';
+/* eslint-disable import/no-unresolved */
```

Comment:
Do you think it would be clearer (I assume you have only one line of comment instead of two?) ? I link to add the precise one for not having to think at which rule was failing.